src/components/mypage/MyPageView.tsx

```diff
@@ -1,4 +1,8 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
+import { getUserData } from 'libs/api';
+
```

Comment:
```suggestion
import { UserDataProps } from 'types/types';
```

요런식으로 `UserData` 의 props 부분만 정의한 인터페이스를 임포트 해주고